### DISCUSSION TITLE
Versioning and locking without erasure coding

### DISCUSSION THREAD

**Discussion by DerEnderKeks** [minio/minio#14506] [Q&A] [answered]
Hello, I was wondering why Minio allows versioning and other features only with erasure coding enabled. Is there any technical reason for that?

I'm planning to use Minio on an already redundant ZFS pool which will be used for multiple different purposes (including block storage), so I have no real need for the integrated erasure coding.

**Comment by harshavardhana** [comment, marked as answer]
It will be available as part of the deprecation of Gateway @DerEnderKeks tentative date is by June - follow the issue here https://github.com/minio/minio/issues/14331

**Reply by tobiasBora** [reply]
The linked issue is closed. What is the current status now?

**Reply by klauspost** [reply]
Single Node Single Drive was added years ago. See the docs: https://docs.min.io/community/minio-object-store/index.html

**Comment by klauspost** [comment]
We are looking into that as a future option:

> Newer MinIO NAS/Single drive setups will move to single data and 0 parity mode (that will re-purpose the erasure-coded backend used for distributed setups but with 0 parity). This would allow for distributed setup features to be available for single drive deployments as well such as Versioning, ILM Replication and more...

There is no timeframe for release, but it is actively being worked on.
